src/pullRequests.ts

```diff
@@ -159,3 +141,31 @@ export function sortPullRequests(
   }
   return pullRequests
 }
+
+const mapPullRequest = (
+  pr: PullData | Unpacked<PullsListData>
+): PullRequestInfo => ({
```

Comment:
I believe this (`Unpacked`) is not required to allow the user to define the flags for the transformers / have labels be lowercase. 
Maybe I am missing something, could you please describe the main benefit?

Reply:
This is just a minor refactor I did while making the change to store labels in a set. Since there was a duplication of all of this mapping logic, I thought it's reasonable to move it to a function. The Unpacked<T> type is simply used to get the type of a single entry of the return type of the list endpoint, since they differ for some reason.

Reply:
Maybe I overlook it, but I don't see `mapPullRequest` being used twice. 

Could you maybe exclude this change from the PR, not sure yet if we need the `Unpacked` type if we keep the old definition for this here.

Reply:
Oh, my bad, I've forgotten to use it in the other place.